Makefile

```diff
@@ -39,6 +39,18 @@ all: build
 clean:
 	@rm -rf $(API_BUILD_DIR) $(UI_BUILD_DIR) ddl/statik.go statik $(LAKEFS_BINARY_NAME) $(LAKECTL_BINARY_NAME)
 
+check-licenses: check-licenses-go-mod check-licenses-npm
+
+check-licenses-go-mod:
+	go get github.com/google/go-licenses
+	$(GOBINPATH)/go-licenses check ./cmd/$(LAKEFS_BINARY_NAME)
+	$(GOBINPATH)/go-licenses check ./cmd/$(LAKECTL_BINARY_NAME)
+
+check-licenses-npm:
+	go get github.com/senseyeio/diligent/cmd/diligent
+	# The -i arg is a workaround to ignore NPM scoped packages until https://github.com/senseyeio/diligent/issues/77 is fixed
+	$(GOBINPATH)/diligent check -w permissive -i ^@[^/]+?/[^/]+ $(UI_DIR)
```

Comment:
@ozkatz for "`permissive`" here: described [here](https://github.com/senseyeio/diligent/blob/master/license.go#L14) as:
> Open Source software that is made available under "non-copyleft" licenses.  These generally require
> attribution of the included open source and may include other obligations
and you get to pick any other category you like from there